cloudplatform/security/src/main/java/com/sap/cloud/sdk/cloudplatform/security/SecurityContextThreadContextDecorator.java

```diff
@@ -31,9 +30,18 @@ public class SecurityContextThreadContextDecorator implements ThreadContextDecor
     @Override
     public <T> Callable<T> decorateCallable( @Nonnull final Callable<T> callable )
     {
+        try {
+            getClass().getClassLoader().loadClass("com.sap.cloud.security.token.SecurityContext");
+        }
+        catch( final Exception e ) {
```

Comment:
```suggestion
        catch( final ClassNotFoundException e ) {
```

Also, apparently one can [skip instantiation of the class](https://www.baeldung.com/java-check-class-exists#telling-classforname-to-skip-initialization).